frontend/src/components/ViewTrips/add-trip.js

```diff
@@ -62,13 +109,17 @@ function createFormGroup(controlId, formLabel, inputType,
  * Component corresponding to add trips modal.
  *
  * @param {Object} props These are the props for this component:
+ * - db: Firestore database instance.
  * - show: Boolean that determines if the add trips modal should be displayed.
  * - handleClose: The function that handles closing the add trips modal.
- * - userEmail: The current users email.
+ * - refreshTripsContainer: Function that handles refreshing the TripsContainer
+ *        component upon trip creation (Remove when fix Issue #62).
+ * - key: Special React attribute that ensures a new AddTripModal instance is
```

Comment:
For now (until I fix issue #62) I am using the key method to refresh the trips container for the current user upon a new trip. But even after that fix, this key will stay because I want to refresh the modal before opening it up each time. The reason why is because there was a bug where you could hit the 'add collaborator' button, close the modal, then re-open it and there would still be three form inputs for collaborators rather than resetting to just one.